### ISSUE TITLE
Supervised learning - different vector file outputs for binary vs text pretrained vectors

### ISSUE THREAD

**Issue by niloch** [facebookresearch/fastText#424]
During supervised learning, when the pretrained vectors are provided in binary format, the model vectors ( .vec file) only include words seen in the training data.

When using text format pretrained vectors, the model vectors includes all the original words, but the vectors for words not seen in training data are all set to zeros.

Why are the formats different?  
Why are words not seen in training discarded (binary file) or set to zero vectors (text file)?  I was hoping fasttext would be able to use pretrained vectors for out-of-vocabulary words during test time.

binary format training command:

```bash
./fasttext supervised -input data.train -output data.model -minCount 25 -epoch 10 -neg 10 -loss softmax -ws 15 -wordNgrams 2 -dim 300 -pretrainedVectors twitter_bin.vec -bucket 100000
```

**Comment by balaram-bhukya** [comment]
@niloch  If it is not handling the OOV words, then what is the point in using pre trained vector ?? 
what is the advantage of using pretrained vectors in classification problem?

**Comment by niloch** [comment]
Dropping the `minCount` option uses the full vocab of the pretrained vectors.

**Comment by HITlilingzhi** [comment]
Excuse me. I am trying to train vectors with pretrained vectors with .vec file and have dropped the minCount option, but the vectors for words not seen in new training data are still set to zeros. Have you handled this now?
